.github/workflows/conformance-delegated-ipam.yaml

```diff
@@ -0,0 +1,342 @@
+name: Conformance Delegated IPAM
```

Comment:
Now that this workflow is following the same structure as the Multi Pool IPAM workflow, I think we also should hook it up in the Ariane config with its own trigger phrase and add that trigger phrase to the workflow name, e.g.

```suggestion
name: Conformance Delegated IPAM (ci-delegated-ipam)
```

and add the following diff to `.github/ariane-config.yaml`:

```diff
diff --git a/.github/ariane-config.yaml b/.github/ariane-config.yaml
index 95dbc7bd1f1a..d8988d9309bc 100644
--- a/.github/ariane-config.yaml
+++ b/.github/ariane-config.yaml
@@ -30,6 +30,9 @@ triggers:
     workflows:
     - conformance-clustermesh.yaml
     - tests-clustermesh-upgrade.yaml
+  /ci-delegated-ipam:
+    workflows:
+    - conformance-delegated-ipam.yaml
   /ci-e2e-upgrade:
     workflows:
     - tests-e2e-upgrade.yaml
@@ -95,6 +98,8 @@ workflows:
     paths-ignore-regex: (test|Documentation|hubble|USERS.md)/
   conformance-clustermesh.yaml:
     paths-ignore-regex: (test|Documentation|hubble|USERS.md)/
+  conformance-delegated-ipam.yaml:
+    paths-ignore-regex: (test|Documentation|hubble|USERS.md)/
   conformance-ipsec-e2e.yaml:
     paths-ignore-regex: (test|Documentation|hubble|USERS.md)/
   conformance-eks.yaml:

```

This should then allow to trigger the workflow using `/ci-delegated-ipam` comments on PRs. I'm not sure whether we also want to hook it up as part of the `/test` trigger phrase (yet) but reading @joestringer's comment https://github.com/cilium/cilium/issues/34788#issuecomment-2338722420 I'd probably say we don't for now.

Reply:
Maybe the simplest step is to add it into `/test` for now as well, then at least @wedaly should get some good feedback about the stability of the test from many runs. We can revisit the cost/benefit to run this over the coming weeks and tweak the triggers/schedule as we implement #34688. /cc @aanm

Reply:
updated to include the suggested changes

Reply:
updated again to include the new workflow in `/test` as well